editoast/src/views/pagination.rs

```diff
@@ -47,7 +47,7 @@ const DEFAULT_PAGE_SIZE: u64 = 25;
 ///
 /// We named the data field `result` to cope with the old pagination schema which
 /// enforced this name. For new paginated responses, the field name is up to your imagination :)
-#[derive(Debug, Clone, Copy, PartialEq, Eq, Serialize, ToSchema)]
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Serialize, Deserialize, ToSchema)]
```

Comment:
```suggestion
#[derive(Debug, Clone, Copy, PartialEq, Eq, Serialize, ToSchema)]
#[cfg_attr(test, derive(Deserialize))]
```

same for other `Deserialize`